.editorconfig

```diff
@@ -0,0 +1,13 @@
+root = true
+
+[*]
+indent_style = space
+indent_size = 2
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+[*.hs]
+indent_size = 2
```

Comment:
I guess we're already using an indent size of two, so this shouldn't be controversial.